Add Cloud Deploy Automations (#259)
googleforgames/global-multiplayer-demo: README.md

Navigate to the
[agones-deploy-pipeline](https://console.cloud.google.com/deploy/delivery-pipelines/us-central1/agones-deploy-pipeline)
delivery pipeline to review the rollout status. Cloud Build will create a Cloud Deploy release which automatically

> deploys Agones to all of the game server clusters automatically. Agones can be deployed manually to subsequent clusters by clicking on the

Reviewer: This line should read:

```
deploys Agones to all of the game server clusters sequentially, with a (by default) 300s wait in between using [Cloud Deploy Automations](https://cloud.google.com/deploy/docs/automation).
```

Sorry, I wasn't clear in my previous comment (not sure why I can't make the suggestion)

> `promote` button within the Pipeline visualization or by running the following gcloud command:

Reviewer: ```suggestion
```

Navigate to the
[agones-deploy-pipeline](https://console.cloud.google.com/deploy/delivery-pipelines/us-central1/agones-deploy-pipeline)
delivery pipeline to review the rollout status. Cloud Build will create a Cloud Deploy release which automatically

> deploys Agones to all of the game server clusters automatically. Agones can be deployed manually to subsequent clusters by clicking on the

Reviewer: ```
[agones-deploy-pipeline](https://console.cloud.google.com/deploy/delivery-pipelines/us-central1/agones-deploy-pipeline)
delivery pipeline to review the rollout status. Cloud Build will create a Cloud Deploy release which automatically
deploys Agones to all of the game server clusters sequentially, with a (by default) 300s wait in between using [Cloud Deploy Automations](https://cloud.google.com/deploy/docs/automation).
```

I'm thinking we drop the docs below for manual promotion, since it's no longer relevant - WDYT?

Once the build process is complete, navigate to the
[agones-deploy-pipeline](https://console.cloud.google.com/deploy/delivery-pipelines/us-central1/global-game-agones-gameservers)
delivery pipeline to review the rollout status. Cloud Build will create a Cloud Deploy release which automatically
deploys the game server Fleet to the global set of Agones Clusters sequentially using [Cloud Deploy Automations](https://cloud.google.com/deploy/docs/automation), starting with the `asia-east1` region first, and with a (by default) 300s wait in between each region.

> The Fleet can be deployed manually to the next region in the queue by pressing the
> `promote` button within the Pipeline visualization or by running the following gcloud command:

Reviewer: ```suggestion
```

Once the build process is complete, navigate to the
[agones-deploy-pipeline](https://console.cloud.google.com/deploy/delivery-pipelines/us-central1/global-game-agones-gameservers)
delivery pipeline to review the rollout status. Cloud Build will create a Cloud Deploy release which automatically

> deploys the game server to the entire Agones Fleet, starting with the `asia-east1` region first.

Reviewer: ```suggestion
deploys the game server Fleet to the global set of Agones Clusters sequentially using [Cloud Deploy Automations](https://cloud.google.com/deploy/docs/automation), starting with the `asia-east1` region first, and with a (by default) 300s wait in between each region.
```

Something like that to specific.

Then I also think we can drop the manual steps as well, since they aren't relevant anymore. WDYT?
